parboiled-java/src/main/java/org/parboiled/transform/RuleMethodInterpreter.java

```diff
@@ -40,6 +40,7 @@
     private final List<Edge> additionalEdges = new ArrayList<Edge>();
 
     public RuleMethodInterpreter(RuleMethod method) {
+        super(ASM5);
```

Comment:
Is there a reason to use this instead of ASM62?

Reply:
Probably not; could you bump @xuwei-k ?